apollo-gradle-plugin/testProjects/groovy/build.gradle

```diff
@@ -1,16 +0,0 @@
-plugins {
-  id("org.jetbrains.kotlin.jvm")
-  id("com.apollographql.apollo3")
-}
-
-repositories {
-  mavenCentral()
-}
-
-apollo {
-  // Make sure Action<> can be called from Groovy
-  service("main") {
-    // Make sure groovy can find the no-arg overload
-    packageNamesFromFilePaths()
-  }
-}
```

Comment:
Why removing that tests? I think a significant part of the user base is using Groovy and our test coverage there isn't so great.

Reply:
I had the impression that it was not used, because it's not referenced by any test, but maybe I'm missing something?

Reply:
Looks like something that I wrote quickly to verify something and was too lazy to add a test for... I'd be in favor of keeping it and adding a single tests that calls `./gradlew --help` or so, just making sure the build script compiles. I can clean up my own mess and do, just let me know.